.vscode/tasks.json

```diff
@@ -0,0 +1,185 @@
+            },
+            "group": "build"
+        },
+        {
+            "label": "Build and push custom operator image: '${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator:next'",
+            "command": "make docker-build docker-push IMG=\"${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator:next\"",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Install OLM on the cluster",
+            "command": "operator-sdk olm install",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Build and push development bundle",
+            "command": "export BUNDLE_IMG=\"${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator-bundle:next\"; make bundle IMG=${BUNDLE_IMG} -s; make bundle-build bundle-push -s BUNDLE_IMG=${BUNDLE_IMG}",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build"
+        },
+        {
+            "label": "Install operator via OLM",
+            "command": "kubectl apply -f config/manager/controller-namespace.yaml; operator-sdk run bundle ${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator-bundle:next --namespace ${NAMESPACE}; kubectl apply -f config/samples/che_v1alpha1_kubernetesimagepuller.yaml -n ${NAMESPACE}",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build",
+            "options": {
+                "env": {
+                    "NAMESPACE": "kubernetes-image-puller-operator"
+                }
+            }
+        },
+        {
+            "label": "UnInstall operator via OLM",
+            "command": "operator-sdk cleanup kubernetes-imagepuller-operator --namespace ${NAMESPACE}",
+            "type": "shell",
+            "args": [],
+            "problemMatcher": [
+                "$go"
+            ],
+            "presentation": {
+                "reveal": "always"
+            },
+            "group": "build",
+            "options": {
+                "env": {
+                    "NAMESPACE": "kubernetes-image-puller-operator"
+                }
+            }
+        },
+
+        {
+            "label": "Deploy operator",
```

Comment:
Not sure if I did something wrong, but I'm having an issue with the `Deploy operator` task,

After running `Update CR/CRDs` and `Build and push custom operator image` tasks, I tried to deploy to crc by running the `Deploy operator` task:
```
> Executing task: make deploy IMG="${IMAGE_REGISTRY_HOST}/${IMAGE_REGISTRY_USER_NAME}/kubernetes-image-puller-operator:next" -s; kubectl apply -f config/samples/che_v1alpha1_kubernetesimagepuller.yaml -n kubernetes-image-puller-operator <

namespace/kubernetes-image-puller-operator created
customresourcedefinition.apiextensions.k8s.io/kubernetesimagepullers.che.eclipse.org created
serviceaccount/kubernetes-image-puller-operator created
role.rbac.authorization.k8s.io/kubernetes-image-puller-operator created
role.rbac.authorization.k8s.io/leader-election-role created
clusterrole.rbac.authorization.k8s.io/metrics-reader created
clusterrole.rbac.authorization.k8s.io/proxy-role created
rolebinding.rbac.authorization.k8s.io/kubernetes-image-puller-operator created
rolebinding.rbac.authorization.k8s.io/leader-election-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/proxy-rolebinding created
configmap/manager-config created
service/controller-manager-metrics-service created
deployment.apps/kubernetes-image-puller-operator created
servicemonitor.monitoring.coreos.com/controller-manager-metrics-monitor created
kubernetesimagepuller.che.eclipse.org/image-puller created
```

I am getting a `CreateContainerError` error:
![image](https://user-images.githubusercontent.com/83611742/134217388-5d17a76f-111e-4f19-8a5e-087f12702abf.png)

Pod events:
![image](https://user-images.githubusercontent.com/83611742/134217481-f98f795f-f5a4-466a-a871-55c071884ed2.png)

Reply:
Containers:
![image](https://user-images.githubusercontent.com/83611742/134235027-a7acfb93-2d8e-4cfe-94c0-a5d2fbbbb1d6.png)

Logs for kube-rbac-proxy:
```
I0921 19:21:36.748823       1 main.go:190] Valid token audiences:
I0921 19:21:36.750170       1 main.go:262] Generating self signed cert as no cert is provided
I0921 19:21:37.209382       1 main.go:311] Starting TCP socket on 0.0.0.0:8443
I0921 19:21:37.213735       1 main.go:318] Listening securely on 0.0.0.0:8443
```

I'm not sure why this is happening, any ideas?

Reply:
Yes, you are right, You found a bug, I will fix it.